Notepad/HomeViewController.swift

```diff
@@ -12,25 +12,38 @@ class HomeViewController: UIViewController {
     @IBOutlet var bodyTextView: UITextView!
     @IBOutlet var titleTextField: UITextField!
     
+    let title_key = "Title"
+    let body_key = "Body"
     override func viewDidLoad() {
         super.viewDidLoad()
     }
```

Comment:
Unrelated to this PR, but do you know what `viewDidLoad()` is / does?

Reply:
it runs when the view controller runs and it loads the view